Dodanie WelcomeViewPage, obrazka btnimg.png do folderu Resources i cz…

dzaleski opened the pull request:
Niżej zamieszczam screen z wykonanego widoku
![image](https://user-images.githubusercontent.com/39300625/79008612-0fc60780-7b5e-11ea-96ed-fc86ed71fe09.png)

dzaleski committed:
Dodanie WelcomeViewPage, obrazka btnimg.png do folderu Resources i czcionek do folderu Assets

Jerry-BloodBerry committed:
Poprawki - nazwy namespace i class

Widzę, że nazwę widoku nadałeś poprawną, ale potem zacząłeś używać innej (polskiej). Żeby nie robić bałaganu to wszystkie nazwy dajemy po angielsku. Trochę nie wiem dlaczego to działało, bo namespace polski w ogóle nie miał metody InitializeComponent() i VS od razu mi to podpowiedział. Dziwne, że nie prowadziło to do błędu kompilacji. Ten commit poprawia te rzeczy.

dzaleski committed:
Merge pull request #74 from MikolajczukKamil/zmiany_w_nazewnictwie

Poprawki - nazwy namespace i class

dzaleski committed:
Zmiana

dzaleski committed:
Poprawiony wizmniejszenie przestrzeni między nagłówkilowkiem a guzikami

dzaleski committed:
Merge branch 'Daniel-Zaleski-WidokPowitalny' of https://github.com/MikolajczukKamil/inwentaryzacja into Daniel-Zaleski-WidokPowitalny

dzaleski committed:
Poprawiony widok: zmniejszenie przestrzeni miedzy naglowkiem a guzikami

dzaleski committed:
Poprawa marginesów elementów

dzaleski commented:
![image](https://user-images.githubusercontent.com/39300625/79041954-11420f00-7bf4-11ea-8f05-8e365eb2a4bb.png)
Zmieniłem urządzenie w emulatorze i wydaje mi się, że teraz widok powinien być poprawnie generowany.

MikolajczukKamil commented:
- w katalogu projektu będzie katalog views i w nim poszczególne widoki, pozwoli to utrzymać przejrzystą strukturę, trzeba przenieść widok do views/view-welcome/
- Dobrze by było to nieco powiększyć tak aby było wyraźne jak w projekcie i może przycisk dolny trochę wyżej bo jak mam zaokrąglenie w rogu ekranu dziwnie to wygląda
- dobrze że jest formatowanie kodu elementy są czytelne ale zwrócę uwagę na jedną rzecz ilość znaków w lini, powinno zrobić się tak aby linia mieściła się na ekranie, często się tu przyjmuje umowną granicę 80 znaków ale nie chodzi o to aby chodzić z linijką i mierzyć po prostu jak jest kilka parametrów to można je podać w osobnych liniach tak jak na górze

![ok](https://user-images.githubusercontent.com/25321219/79047266-da7cf080-7c15-11ea-98c9-8a9dd870947c.PNG)

dzaleski committed:
Poprawa widoku: powiększenie elementów i czcionki

dzaleski commented:
Screen widoku po zmianie wielkości
![image](https://user-images.githubusercontent.com/39300625/79049593-ce4c5f80-7c24-11ea-8754-3350df61a6fa.png)

Jerry-BloodBerry committed:
Inny margines dla labela

Czy taki margines nie będzie lepszy? Z takim jak był to na Pixel2 mi się rozjechało. Teraz przetestowałem na 3 różnych telefonach i było ok. Telefony na których testowałem to Pixel 2, Galaxy Nexus i Xiaomi Redmi 6

dzaleski committed:
Merge pull request #76 from MikolajczukKamil/poprawka-header-powitanie

Usunięcie lewego i prawego marginesu dla nagłówka

Jerry-BloodBerry reviewed (approved):
Teraz jest już bardzo dobrze, tak jak w projekcie, można mergować 👍

MikolajczukKamil commented:
Jak byś jeszcze wsadził to w views/view-welcome/, byłby miodzio :D

dzaleski committed:
Dodanie Welcome View do folderu view/view-welcome/

dzaleski committed:
Merge branch 'Daniel-Zaleski-WidokPowitalny' of https://github.com/MikolajczukKamil/inwentaryzacja into Daniel-Zaleski-WidokPowitalny

dzaleski commented:
Nie mogłem znaleźć folderu views/ w projekcie więc go stworzyłem i dodałem tam wykonaną stronę. Nie do końca wiem czy o to chodziło.

MikolajczukKamil reviewed (approved):
Jest miodzio tylko konflikty się narobiły

dzaleski committed:
Revert "Dodanie Welcome View do folderu view/view-welcome/"

This reverts commit 5f65e5ceee1afe763d64b4d378e77052656122c8.

dzaleski committed:
Dodanie strony do folderu views/welcome-view/

dzaleski committed:
rozwiazanie konfliktu
